docs/development/new-cloud-provider.md

```diff
@@ -0,0 +1,57 @@
+# Adding Cloud Providers
+This document provides an overview of how to integrate a new cloud provider into Gardener. Each component that requires integration has a detailed description of how to integrate it and the steps required.
+
+## Cloud Components
```

Comment:
space after header

Reply:
If you insist. Not sure why though? markdown automatically eliminates extra newlines there.

Reply:
Done